types/auth0/index.d.ts

```diff
@@ -67,8 +80,7 @@ export interface Identity {
   isSocial: boolean;
 }
 
-export interface UpdateUserParameters {
-  id: string;
+export interface UpdateUserParameters extends ObjectWithId {
```

Comment:
TypeScript uses a structural type system, so `UpdateUserParameters` is equivalent to `ObjectWithId` here since it doesn't declare any more members.